examples/09_Original_MAD.ipynb

```diff
@@ -69,15 +68,15 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/billbrod/Documents/plenoptic/plenoptic/metric/perceptual_distance.py:92: UserWarning: dynamic_range is 1 but image range falls outside [0, 1] img1: tensor([0.0039, 1.0000]), img2: tensor([-11.9009,  15.4622]). Continuing anyway...\n",
+      "/home/billbrod/Documents/plenoptic/plenoptic/metric/perceptual_distance.py:92: UserWarning: dynamic_range is 1 but image range falls outside [0, 1] img1: tensor([0.0039, 1.0000]), img2: tensor([-10.7873,  12.8881]). Continuing anyway...\n",
```

Comment:
Maybe mention this is a work in progress? 
Add text/headers etc to give user an idea of what's going on.